caddyhttp/errors/setup_test.go

```diff
@@ -85,13 +85,14 @@ func TestErrorsParse(t *testing.T) {
 				Roller: httpserver.DefaultLogRoller(),
 			},
 		}},
-		{`errors errors.txt { rotate_size 2 rotate_age 10 rotate_keep 3 }`, false, ErrorHandler{
+		{`errors errors.txt { rotate_size 2 rotate_age 10 rotate_keep 3 rotate_compress true }`, false, ErrorHandler{
```

Comment:
Let's get rid of "true" here, since the presence of "rotate_compress" will make that obvious. (And lack of it is obviously "false".) In other words, the line would be "rotate_compress" by itself, no arguments.